tslint.json

```diff
@@ -112,7 +112,6 @@
               "singleline": "never"
           }
       ],
-      "space-before-function-paren": true,
```

Comment:
can you make ts lint a blocking step in the build so that travis ci will fail when ts lint fails?

I think you can add it to package.json

```
"scripts": {
    "build": "tslint && ..."
}
```

Reply:
I think it makes more sense to make it a step in the jenkins job. I'll add it there

Reply:
If it's only a step in travis-ci then a developer may not see a tslint error until it hits ci